lib/parser/parser-utils.ts

```diff
@@ -4,7 +4,7 @@
 
 import * as path from 'path';
 import * as htmlparser from 'htmlparser2';
-const merge = require('lodash.merge');
+import merge from 'lodash.merge';
```

Comment:
For this change I had to enable `esModuleInterop` in the tsconfig.json file.

Reply:
Not sure if this is a good idea. It might cause all downstream libraries to also enable `esModuleInterop`.

You can remove that setting and instead do `import * as merge from 'lodash.merge';`

Reply:
See for example this comment: https://github.com/apollographql/apollo-server/issues/1977#issuecomment-683235089

Reply:
I had to fix it a bit differently but `import merge = require('lodash.merge');` fixed the problem.